src/js/models/geocoder/GeocoderSearch.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+define(
+  [
+    'underscore',
+    'backbone',
+    'models/geocoder/GoogleMapsGeocoder',
+    'models/geocoder/GoogleMapsAutocompleter',
+    'models/geocoder/GeocodedLocation',
+    'models/geocoder/Prediction',
+  ],
+  (
+    _,
+    Backbone,
+    GoogleMapsGeocoder,
+    GoogleMapsAutocompleter,
+    GeocodedLocation,
+    Prediction,
+  ) => {
+    /**
+    * @class GeocoderSearch
+    * @classdes GeocoderSearch interfaces with various geocoding and location
+    * searching services.
+    * @classcategory Models/Geocoder
+    */
+    const GeocoderSearch = Backbone.Model.extend({
```

Comment:
I haven't seen any places where plan classes are used, though I do agree, this probably warrants more discussion.. @robyngit for any thoughts you may have

Reply:
hmmm yea- if not `class`, prototypes without Backbone Model exists, ex. Utilities.js. Though i don't see why we can't just use a class.

Reply:
Could you see us using some of the backbone model functionality as we add different geocoding services in the future, for example, listening to when a new geocoder model is added as an attribute to this model?

Reply:
I don't think it will ever need any of the Model capabilities. I'll migrate to a JS `class`, where should it live though? If it's not a model, I think I should move it out of js/models, is js/geocoder fine to create? Or should I put it next to Utilities.js in js/common?